modules/fundamental/src/sbom/endpoints/test.rs

```diff
@@ -1502,3 +1502,53 @@ async fn all_labels(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
 
     Ok(())
 }
+
+#[test_context(TrustifyContext)]
+#[test(actix_web::test)]
+async fn get_cbom(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
+    let app = caller(ctx).await?;
+    let id = ctx
+        .ingest_document("cyclonedx/cryptographic/cbom.json")
+        .await?
+        .id
+        .to_string();
+    let uri = format!("/api/v2/sbom/{id}");
```

Comment:
@desmax74 Can we also call `/packages` call here and loop through them, just to make sure there's no any errors coming from the backend in this case?